### ISSUE TITLE
not working when using symfony dkim-signer

### ISSUE THREAD

**Issue by s4mpl3d** [zenstruck/mailer-test#23]
### unsigned
```php
$email = new Email();
$html = $this->twig->render('test.html.twig',[]);
$email->to($user->getEmail());
$email->subject("test");
$email->html($html);
$email->from('noreply@test.com');
$this->mailer->send($email);
```
`$this->mailer()->sentEmails()` => contains an email

### signed
```php
$email = new Email();
$html = $this->twig->render('test.html.twig',[]);
$email->to($user->getEmail());
$email->subject("test");
$email->html($html);
$email->from('noreply@test.com');

$signer = new DkimSigner($this->dkimPrivateKey, $this->dkimDomain, $this->dkimSelector);
$signedEmail = $signer->sign($email);
$this->mailer->send($signedEmail);
```
`$this->mailer()->sentEmails()` => contains no email

https://symfony.com/doc/current/mailer.html#dkim-signer

**Comment by kbond** [comment]
Hey @s4mpl3d, thanks for the report.

I've re-created the issue in #24 but am having some issues finding a solution.

The crux of the problem is calling `$signer->sign($email)` on an `Email` instance converts it to a `Message`. This package was only looking for `Email`'s so `Message`'s were excluded.

#24 converts `Message`'s back into `Email`'s but we lose the html/text/attachments. They are technically _in the_ `Email` but not in the way this package expects (raw data).

I think a solution could be to convert the raw parts back into attachments.

**Comment by kbond** [comment]
Fixed and released in 1.3.0. Let me know if it doesn't work for you.

**Comment by s4mpl3d** [comment]
@kbond Thank you very much for your effort and fast response! All my test's are green again
